services/api/service_test.go

```diff
@@ -410,6 +415,415 @@ func TestBuilderSubmitBlockSSZ(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestSubmitTobTxs(t *testing.T) {
```

Comment:
too long of a test. can you do a TDT here ? looks like a lot of repeated code: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests